janet/janet-bundler/Portfile

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           janet  1.0
+
+name                janet-bundler
+github.setup        joy-framework bundler 23785e59c468d39973dfc9a2aeee8e1b135f15fd
+# from project.janet
+version             0.1.0
+revision            0
+categories          janet devel
+license             MIT
+maintainers         nomaintainer
+platforms           darwin
+description         A very simple css and js bundler for janet
+long_description    {*}${description}
+
+checksums           rmd160  fd7ea7ec29852b9cb16e778d9aeae4ffb5ae9377 \
+                    sha256  aeea6442314562f6ee9136d16c17266064bf49b2af95851d271070242ac79b62 \
+                    size    1861
+
+depends_lib         port:janet-path \
```

Comment:
Can you always use -append when adding dependencies. It avoids potential future bugs if a portgroup in the future adds dependencies itself, as without -append you would inadvertently remove them.

Reply:
I agree that `-append` is safer and causes no harm.
I will look into making the change.
Ideally, port groups should be designed so this is not necessary.

Reply:
Its not possible to avoid portgroups sometimes having to add dependencies. E.g. the legacy support PG has to decide if or not a port needs a dependency on the port or not. There are other examples I am sure.

Reply:
Requiring `-append` may indeed sometimes be necessary.
I will make the change to `-append` just to be safe.
My point is only that dependencies should be added by `port::register_callback` in port groups whenever possible (sometimes it is not).